### ISSUE TITLE
Rename rooms

### ISSUE THREAD

**Issue by MatthewSteen** [gitterHQ/gitter#1162]
This feature would offer additional flexibility to the room settings menu.

**Comment by MadLittleMods** [comment]
If a room is based off of a repository, we will automatically pick up the rename and move it over.

For what we call channel rooms, not associated with a repo, we would need to add this functionality to allow the user to rename their room. In the meantime, if you need to rename a room, create the new room and send us a message support@gitter.im with the old and new room names and we can move over the messages manually. Note that this won't move the users over.

**Comment by MadLittleMods** [comment]
@hugovk no, please create a new issue for unrelated things. I understand I mentioned being able to transfer messages manually but you could reference the comment in your new issue.
